release_notes/topics/18_0_0.adoc

```diff
@@ -1,5 +1,22 @@
 = Highlights
 
+== New Operator tech-preview
+
+With this release, we're introducing a brand new {project_operator} as a tech-preview. Apart from being rewritten from
+scratch, the main user-facing change from the legacy Operator is the used {project_name} distribution – the new Operator
+uses the Quarkus distribution of {project_name}. With that, there are a few breaking changes. For details, incl.
+installation and migration instructions, see the https://www.keycloak.org/guides[Operator related guides].
```

Comment:
```suggestion
installation and migration instructions, see the https://www.keycloak.org/guides#operator[Operator related guides].
```

Reply:
Adding the fragment scrolls directly to the section, see https://www.keycloak.org/guides#server